docs/backup-restore.md

```diff
@@ -90,7 +90,8 @@ Specifically, please follow the steps below.
    * Remove table A
    * Create a table named A with backup B
 
-Note that tables can only be restored one by one so you need to do the above steps for each table.
+Note:
 
-Note also that some of the configurations such as PITR and auto scaling policies are reset to the default values for restored tables, so please manually configure required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
-If you used the schema loader and didn't change any other configurations, executing the schema loader with the same schema file is one of the ways to restore the original configuration. It only sets missing configurations if a specified table exists.
+* You can create a backup for the `scalardb.metadata` table without restoring the PITR because it contains metadata of the schemas (It will not change after the schema creation).
```

Comment:
I don't get what it means.
What do you mean by "create a backup ... without restoring the PITR"?

Also, as I said many times, what do you mean by `You can`. You should? or you must? or what?

Reply:
@feeblefakie san,
> I don't get what it means.
What do you mean by "create a backup ... without restoring the PITR"?

By default, the schema tool will not enable PITR for `scalardb.metadata` so we can't do the following procedures to create a backup for this table.

```
1. Create a backup
   * Select the mid-time of paused duration as the restore point.
   * Restore with PITR of table A to another table B
   * Take a backup of the restored table B (assume the backup is named backup B)
   * Remove table B 
```
In this case, we can create a backup without having to restore with PITR.

Do we need to add this step? 
If it needs to keep, I will update it with `You should`.

Reply:
Sorry, I don't get it. Maybe English issue.
I don't get what `creating a backup without restoring ...` means. It sounds very wrong or misexpressed.

> By default, the schema tool will not enable PITR for scalardb.metadata so we can't do the following procedures to create a backup for this table.

The schema tool will enable PITR for other tables?

Reply:
By default, the schema tool will not enable PITR for `scalardb.metadata`, so we can create back using the `Create backup` button of the following screen. 
<img width="824" alt="Screen Shot 2022-05-10 at 9 51 26 AM" src="https://user-images.githubusercontent.com/58680313/167521372-001b661c-8949-4923-abcd-4cfede82e163.png">
Which means we do not need to follow the following procedures for creating a backup for `scalardb.metadata`.
```
1. Create a backup
   * Select the mid-time of paused duration as the restore point.
   * Restore with PITR of table A to another table B
   * Take a backup of the restored table B (assume the backup is named backup B)
   * Remove table B 
```

Reply:
>The schema tool will enable PITR for other tables?

yes

Reply:
> Creating a backup and restoring it are two separate things.

```
1. Create a backup
   * Select the mid-time of paused duration as the restore point.
   * Restore with PITR of table A to another table B
   * Take a backup of the restored table B (assume the backup is named backup B)
   * Remove table B 
 ```
While we creating a backup from PITR, first we need to restore tables with PITR (`Restore with PITR of table A to another table B.`)

Reply:
As we discussed, later versions of the schema tool will enable PITR for the metadata table.
So I removed the unclear sentence.